static/js/draw12.js

```diff
@@ -1,381 +0,0 @@
-        });
-
-
-
-        Poly.setMap(map);
-        isClosed=true;
-  
-
-    }
-    else{
-
-        if(PointArray.length==0)
-        {
-           drawMap(map);
-
-        }
-        else
-        {
-            Poly = new google.maps.Polygon({
-            paths:PointArray,
-            draggable: true,
-            editable: true,
-            strokeColor: '#FF0000',
-            strokeOpacity: 0.8,
-            strokeWeight: 2,
-            fillColor: '#FF0000',
-            fillOpacity: 0.35
-            });
-        
-            Poly.setMap(map);
-            isClosed=true;
-        }         
-    }
-
-}
-
-function laodmapcity(){
-    
-    var id = django.jQuery("#id_city option:selected").val();  
-    $.ajax({
-        url : "/admin/Acitymapdisplay/",
-        type : "POST",
-       data : { 'id' : id},
-  
-        contenttype : "json",
-        success : function(json) {
-            PolygonPoints=Poly.getPath().getArray();
-            initMap(json);
-        },
-        async:false
-    });
-   
-}
-
-
-
-function laodmapcity2(){
-    var id = django.jQuery("#id_city option:selected").val();  
-    $.ajax({
-        url : "/admin/Acitymapdisplay/",
-        type : "POST",
-       data : { 'id' : id},
-  
-        contenttype : "json",
-        success : function(json) {
-          val = json;
-        },
-        async:false
-    });
-
-    return val;
-}
-
-
-function drawMap(){
-  
-      isClosed = false;
-      poly = new google.maps.Polyline({ map: map, path: [], strokeColor: "#FF0000", strokeOpacity: 1.0, strokeWeight: 2 });
-  
-      google.maps.event.addListener(map, 'click', function (clickEvent) {
-        if (isClosed)
-            return;
-        var markerIndex = poly.getPath().length;
-        var isFirstMarker = markerIndex === 0;
-        var Point = clickEvent.latLng;
-        PointArray.push(Point); 
-
-        var marker = new google.maps.Marker({ map: map, position: clickEvent.latLng, draggable: true });
-        markersArray.push(marker);
-        if (isFirstMarker) {
-            google.maps.event.addListener(marker, 'click', function () {
-                if (isClosed)
-                    return;
-                var path = poly.getPath();
-                poly.setMap(null);
-                poly = new google.maps.Polygon({ map: map, path: PointArray, strokeColor: "#FF0000", strokeOpacity: 0.8, strokeWeight: 2, fillColor: "#FF0000", fillOpacity: 0.35,draggable: true,
-            editable: true });
-                isClosed = true;
-                clearOverlays();
-            });
-        }
-
-        google.maps.event.addListener(marker, 'drag', function (dragEvent) {
-            poly.getPath().setAt(markerIndex, dragEvent.latLng);
-        });
-        poly.getPath().push(clickEvent.latLng);
-    }
-       ); 
-
-}
-  
-function clearOverlays() {
-  for (var i = 0; i < markersArray.length; i++ ) {
-    markersArray[i].setMap(null);
-  }
-  markersArray.length = 0;
-}
```

Comment:
Add comments to each of the functions and see that you have single document ready method